std/fs/expand_glob.ts

```diff
@@ -62,6 +62,12 @@ function comparePath(a: WalkEntry, b: WalkEntry): number {
 /**
  * Expand the glob string from the specified `root` directory and yield each
  * result as a `WalkEntry` object.
+ *
+ * Examples:
+ *
+ *     for await (const file of expandGlob("**\/*.ts")) {
+ *       console.log(file.path);
```

Comment:
```suggestion
 *       console.log(file);
```
would match the sync example. Provides better insight in the output.